doc/complete-reference.md

```diff
@@ -84,8 +84,28 @@ Nelmio\Entity\User:
         favoriteNumber: 42
 ```
 
-To go further we can just randomize data.
+You can refer to previous fixtures to get the same amount of fixtures.
+The current formatter is set to the referenced fixture.
 
+```yaml
+Nelmio\Entity\UserDetail:
+# reference by concrete id
+    user{@user1}:
```

Comment:
I'm sorry I just find this a bit confusing, could you explain what the result is?